vtsuperdarn/davitpy
# UAF multi-channel data read radDataTypes

It seems as though we lost the ability to read in all of the UAF multi-channel data with just one call.  So, if no
channel is specified, then find all of the channels.

To test this:
1) on the develop branch, try to read in data with a radcode of kod from 20140624 and no channel set.  This will not find any files.
2) using the pull request here, try the same thing and this will file both kod.c and kod.d files.
3) using the pull request here, keep radcode and the date the same, but set channel = c and see that only kod.c files load.
4) Then remove the data in the DAVIT_TMPDIR and rerun the load with channel = c.
5) Then remove the channel variable setting and see that both kod.c and kod.c files load without using the previously cached kod.c file.
6) 
7) Profit!

Cheers,

-Kevin
